proknow-sdk/ProKnowApi.cs

```diff
@@ -1,150 +1,155 @@
-﻿using Microsoft.Extensions.Logging;
-using ProKnow.Collection;
-using ProKnow.Exceptions;
-using ProKnow.Patient;
-using ProKnow.Role;
-using ProKnow.Scorecard;
-using ProKnow.Upload;
-using ProKnow.User;
-using System;
-using System.IO;
-using System.Text.Json;
-using System.Threading.Tasks;
-
-namespace ProKnow
-{
-    /// <summary>
-    /// This is the main class that should be instantiated during application startup with your base URL (which should
-    /// include your account subdomain) and your API token credentials
-    /// </summary>
-    public class ProKnowApi : IProKnowApi
-    {
-        private readonly ILogger _logger;
-
+﻿using Microsoft.Extensions.Logging;

```

Comment:
I'm confused why there are so many changes detected in this file. Did the line endings change or something?

Reply:
This has been resolved